app/scripts/metamask-controller.js

```diff
@@ -314,18 +308,15 @@ module.exports = class MetamaskController extends EventEmitter {
    * @returns {Object} status
    */
   getState () {
-    const wallet = this.configManager.getWallet()
     const vault = this.keyringController.store.getState().vault
-    const isInitialized = (!!wallet || !!vault)
+    const isInitialized = !!vault
 
     return {
       ...{ isInitialized },
       ...this.memStore.getFlatState(),
-      ...this.configManager.getConfig(),
```

Comment:
I found this line to be particularly egregious, because it always returned an empty object 😔

Reply:
🤦‍♂️